### ISSUE TITLE
Faire un thème pour l'application

### ISSUE THREAD

**Issue by Hamtara** [ApplETS/SEEMobile-Ionic#9]
- Changer le thème par défaut
- Améliorer les vues suivant la plateforme

**Comment by willkoua** [comment]
est ce que tu as déjà un thème en tête?

**Comment by Hamtara** [comment]
Oui, j'avais commencé avec les couleurs des en-tête et tout mais c'est améliorable.

**Comment by willkoua** [comment]
Ok, j'imagine que la couleur de bas va être du rouge ?
